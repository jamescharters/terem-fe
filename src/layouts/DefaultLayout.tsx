import { Outlet } from "react-router-dom";
import PageStyles from "./Default.module.scss";

export const DefaultLayout: React.FC<React.PropsWithChildren> = () => {
    return (
        <div className={PageStyles.container} data-testid="page">
            <Outlet />
        </div>
    );
};
