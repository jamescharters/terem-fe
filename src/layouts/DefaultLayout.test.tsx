import { render } from "@testing-library/react";
import { describe, it, expect } from "vitest";
import { DefaultLayout } from "./DefaultLayout";

describe("DefaultLayout", () => {
    it("renders the component", () => {
        const result = render(<DefaultLayout />);

        expect(result.asFragment()).toMatchSnapshot();
    });
});
