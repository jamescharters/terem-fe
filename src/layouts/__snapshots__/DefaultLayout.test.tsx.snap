// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`DefaultLayout > renders the component 1`] = `
<DocumentFragment>
  <div
    class="_container_98e832"
    data-testid="page"
  />
</DocumentFragment>
`;
